tools/dump_kubernetes.sh

```diff
@@ -246,9 +277,18 @@ dump_kubernetes_resources() {
 
   mkdir -p "${OUT_DIR}"
   # Only works in Kubernetes 1.8.0 and above.
-  kubectl get --all-namespaces --export \
-      all,jobs,ingresses,endpoints,customresourcedefinitions,configmaps,secrets,events \
-      -o yaml > "${RESOURCES_FILE}"
+  if [ -z "${namespaces}" ]; then
+    kubectl get --all-namespaces --export \
```

Comment:
`Flag --export has been deprecated, This flag is deprecated and will be removed in future`

Reply:
I've seen this error message too, I did not consider removing it for backward compatibility for this change, do you think we should take this out in this PR?